index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Please fill out this survey</title>
+    
+</head>
+
+<body>
+    <header id="head">
+        <h1 id="title">Survey Form</h1>
+        <p id="description">This is a description</p>
+    </header>
+    <form action="/backend.php" id="survey-form">

```

Comment:
- [ ] It would be better to wrap the `<form>` inside of a `<section>` or `<main>` to correctly make use of semantic HTML tags.